MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+recursive-include openfisca_web_api/i18n *.mo
+include *.md
+include LICENSE
+include *.ini
```

Comment:
Aren't those files already covered by [those lines](https://github.com/openfisca/openfisca-web-api/blob/d790243123c72fd198cc29a63d6dd4dedb316510/setup.py#L27-L39)?

I guess that if this file does not exist, the RPM package won't contain the mentioned files

My worry here is to duplicate the places the information is stored (`setup.py` and `MANIFEST.in`). But that's perhaps the only way to do(?)

I'd like to understand why it works now with building wheels, but not RPMs.

I think we should stick with https://github.com/pypa/sampleproject/ and it seems to duplicate the information too.